src/Domain/Person.php

```diff
@@ -170,8 +172,15 @@ public function validateCaptchaAndRawPasswordSetIfNew(ExecutionContextInterface
         }
 
         // Entity brand new or somehow otherwise without a password, and none set.
-        if (empty($this->password) && empty($this->raw_password)) {
-            $context->buildViolation('Password is required to create an account')
+        $passwordMissing = (
+            empty($this->password) &&
+            (empty($this->raw_password) || mb_strlen($this->raw_password) < static::MIN_PASSWORD_LENGTH)
```

Comment:
@NoelLH I think we should add some extra `()` to explicitly highlight if the `||` operator applies to `empty($this->password) && (empty($this->raw_password)` as a whole or just `(empty($this->raw_password)`.

Reply:
@alihejazi I don't understand where these would go. To me it is already very clear and extraneous brackets would make it harder to read. There are already no dependencies on operator precedence, the brackets around the second condition line make it explicit.